### ISSUE TITLE
Update RNPM for RN 0.60+

### ISSUE THREAD

**Issue by kidx5452** [Sunhat/react-native-extra-dimensions-android#56]
Project need add file react-native.config.js to update "rnpm" props in packages.json.

**Comment by TheChanDI** [comment]
yeahh.. i am not being able to use this library because of RN 0.60+. Please update 'rnpm' props in packages.json

**Comment by janczizikow** [comment]
@Sunhat would it be possible to release new version of the package to the npm?

**Comment by andreifilip123** [comment]
Any upcoming release ? Or we should use the master branch ?

**Comment by iamandiradu** [comment]
Release?

**Comment by nathantaal** [comment]
Release?

**Comment by christheyounger** [comment]
Release?

**Comment by a-tokyo** [comment]
Can you please Release? 💪

**Comment by Sunhat** [comment]
Dead project. Nobody should use this.

**Comment by orafal-dev** [comment]
@Sunhat any recommendations on alternatives?

**Comment by ziyaddin** [comment]
@rafalolszewski94 have you found any alternatives?

**Comment by a-tokyo** [comment]
@ziyaddin I use this for now https://github.com/pluralcom/react-native-extra-dimensions-android

```js
{
    "react-native-extra-dimensions-android": "https://github.com/pluralcom/react-native-extra-dimensions-android",
}
```

**Comment by Sunhat** [comment]
If Airbnb do not need this, then I'm going to say nobody else does either

**Comment by Sunhat** [comment]
Please provide me with an argument as to why this is necessary?

**Comment by a-tokyo** [comment]
Example usecase:
```js
  if (Platform.OS === 'android') {
    const ExtraDimensions = require('react-native-extra-dimensions-android');
    const STATUSBAR_HEIGHT = ExtraDimensions.get('STATUS_BAR_HEIGHT');
    const WINDOW_HEIGHT =
      ExtraDimensions.get('REAL_WINDOW_HEIGHT') -
      STATUSBAR_HEIGHT -
      ExtraDimensions.get('SOFT_MENU_BAR_HEIGHT');
    return { STATUSBAR_HEIGHT, WINDOW_HEIGHT };
  }
```

NOTE: Different existing RN projects ALREADY use this repo, which means that upgrading RN would be an issue just because of RNPM issues that can be maintained.

**Comment by christheyounger** [comment]
Yes, I fail to see why accepting a PR and creating a release tag is too big a burden.
The package is otherwise functional, just needs an occasional update that others in the community can provide.
You just need to create a tag.

**Comment by rishabh876** [comment]
> @ziyaddin I use this for now https://github.com/pluralcom/react-native-extra-dimensions-android
> 
> ```js
> {
>     "react-native-extra-dimensions-android": "https://github.com/pluralcom/react-native-extra-dimensions-android",
> }
> ```

This is not working. Since in package.json has `files` defined and it does not have `react-native.config.js` in it. So npm does not fetch `react-native.config.js` file

**Comment by a-tokyo** [comment]
If anyone is using https://github.com/pluralcom/react-native-extra-dimensions-android and would like us to publish an NPM release instead of just github, please let me know 👍🏻 

NPM is more reliable since using github link simply refers to the latest version and not an exact version
